CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.21.0 (2023-12-06)
+
+### New features
+
+- A new sender adaptor `require_started` allows to detect unstarted senders. ([#869](https://github.com/pika-org/pika/pull/869))
+- The conversion from `any_sender` to `unique_any_sender` has been optimized, reusing the same
+  storage. ([#844](https://github.com/pika-org/pika/pull/844))
+- Limit the number of streams created in `cuda_pool`. ([#864](https://github.com/pika-org/pika/pull/864))
+
+
+### Breaking changes
+
+- Some runtime initialization functions `pika::start` and `pika::finalize` now return `void`. ([#825](https://github.com/pika-org/pika/pull/825))
+
+### Bugfixes
+
+- One lifetime bug in split has been fixed. ([#839](https://github.com/pika-org/pika/pull/839))
+- Fixes spinlock deadlock detection when suspension is disabled in `yield_while`. ([#856](https://github.com/pika-org/pika/pull/856))
```

Comment:
```suggestion
- One lifetime bug in `split` has been fixed. ([#839](https://github.com/pika-org/pika/pull/839))
- Fixes spinlock deadlock detection when suspension is disabled in `yield_while`. ([#856](https://github.com/pika-org/pika/pull/856))
```

Could you reword the second point to be similar in style to the other points as well? Something in the direction of "`yield_while` now blah...`.